dhcp/dhcp.py

```diff
@@ -583,6 +583,19 @@ def sorted_hosts(hosts):
 IP    = b'\x45\x00\x00\x00\x00\x00\x40\x00\x40\x11\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00'
 ETHER = b'\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x08\x00'
 
+def getmac():
+    try:
+        yamlFile = open('../piman.yaml')
```

Comment:
relative paths are tricky, but ../ is even trickier. self.configuration already has this info.

Reply:
So I have taken a look at the self.configuration but I am unclear how it contains the information about the location of the piman.yaml file.